be/src/http/http_common.h

```diff
@@ -37,6 +37,7 @@ static const std::string HTTP_STRICT_MODE = "strict_mode";
 static const std::string HTTP_TIMEZONE = "timezone";
 static const std::string HTTP_EXEC_MEM_LIMIT = "exec_mem_limit";
 static const std::string HTTP_EXEC_JSONPATHS  = "jsonpaths";
+static const std::string HTTP_EXEC_JSONROOT  = "json_root";
```

Comment:
```suggestion
static const std::string HTTP_JSONROOT  = "json_root";
```

EXEC is not some kind of prfix, it is with `EXEC_MEM_LIMIT`.
Also modify `HTTP_EXEC_JSONPATHS` to `HTTP_JSONPATHS`

Reply:
ok